### PR TITLE
feat(nvidia): Attempt to support NVIDIA open

### PR DESCRIPTION
**Pull request by EyeCantCU**

### CODE REVIEW

**Review by akdev1l — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -25,13 +25,22 @@ jobs:
         cfile_suffix: [common, nvidia]
         major_version: [37, 38, 39]
         nvidia_version: [0, 470, 535]
+        nvidia_varient: [none, proprietary, open]
```
[the remark this answers is not in the document]
Comment (EyeCantCU) [reply .github/workflows/build.yml]: Corrected :)

**Comment by dylanmtaylor** [comment]
Is this PR abandoned? If this is still desired, I could try to get this to completion.

**Comment by dylanmtaylor** [comment]
I think this should be reworked on top of the negativo17 change that is currently in progress

**Comment by p5** [comment]
Superseded by #220

**Pull request closed without merging**